vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1318,6 +1320,9 @@ void AdsImpl::SustainAdNotificationInteractionIfNeeded(
 
   BLOG(1, "Sustained ad for " << url);
 
+  AppendAdNotificationToHistory(last_shown_ad_notification_,
```

Comment:
@tmancey just to make sure I understand the change - was the problem that we forgot to append the landed confirmation to history so the existing frequency cap wouldn't yield the desired result and appending it now fixes the frequency cap behaviour w/o changing its logic?

Reply:
@moritzhaller This is correct